<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #9E9B9B;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignRight';
border: 1px solid gray;
background-color: white;
border-radius:30px;
font-size: 35px;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: red;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sign">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Add">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Multiply">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Subtract">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Equalsto">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: orange;
font-size: 35px;
}

QPushButton:pressed{

background-color: white;
}

</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Decimal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>240</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 60px;
background-color: white;
font-size: 35px;
}

QPushButton:pressed{

background-color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
